### ISSUE TITLE
Feedback about Quickstart

### ISSUE THREAD

**Issue by dearwind153** [pytorch/tutorials#3904]
There is the following issue on this page: https://docs.pytorch.org/tutorials/beginner/basics/quickstart_tutorial.html

In the Section "Optimizing the Model Parameters", the backpropagation code order should be 

        # Backpropagation
        _**optimizer.zero_grad()**_
        loss.backward()
        optimizer.step()

**Comment by akshayavb99** [comment]
I can work on this issue
